modules/platforms/dotnet/Apache.Ignite.Core/IgniteConfiguration.cs

```diff
@@ -72,6 +72,11 @@ public class IgniteConfiguration
         /// </summary>
         public const int DefaultJvmMaxMem = -1;
 
+        /// <summary>
+        /// Default query timeout.
+        /// </summary>
+        public static readonly TimeSpan DefaultQueryTimeout = TimeSpan.FromMilliseconds(0);
```

Comment:
Same name as the property below - won't compile. When the property is `X`, default value for it is `DefaultX`, so in this case it would be `DefaultDefaultQueryTimeout`. But since it is just `default(TimeSpan)`, I would remove it altogether.

Reply:
@ptupitsyn Thank you for reviewing and sharing your feedback. Can you please clarify if you are asking to rename the parameter from `DefaultQueryTimeout ` to `QueryTimeout ` or to remove this parameter from `IgniteConfiguration.cs`

Reply:
I'm asking to remove `public static readonly TimeSpan DefaultQueryTimeout`, because it is zero.